index.rst

```diff
@@ -70,292 +70,140 @@ In gross, the conceptual steps are:
+availability of ``rc1``.
 
-    PRODUCTS: lsst_distrib
-    EUPS_TAG: v42_0_0_rc1
-    SMOKE: true
-    RUN_SCONS_CHECK: true
-    PUBLISH: true
 
-Build and Publish ``scipipe`` docker image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+2nd+ Release Candidate(s)
+-------------------------
 
-https://ci.lsst.codes/blue/organizations/jenkins/release%2Fdocker%2Fbuild-stack/activity
+An ``.rcX``, where X is ``> 1``, is only required if a problem is found with
+the initial ``rc``.
 
-.. code-block::
+**Any subsequent ``rc` differs slightly from the initial ``rc1`` process
+because it inherently is not identical to a previous ``git tag`` (if it was,
+there would be no reason to produce another ``rc``). The creation of a git
+release branch prior to ``rc1`` would eliminate the differences.**
 
-    PRODUCTS: lsst_distrib
-    TAG: v42_0_0_rc1  # eups tag
-    NO_PUSH: false
+Branch, Merge, Tag
+^^^^^^^^^^^^^^^^^^
 
-Build and Publish ``jupyterlab`` docker image
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Any git repository that needs to be modified for additional ``rc`` releases
+should be **branched** and have the necessary changes merged to a release
+branch.  Eg., if changes were needed in ``v42.0.0.rc1`` a "release branch"
+along the lines of ``v42.x`` should be created in the repos that need changes.
 
-https://ci.lsst.codes/blue/organizations/jenkins/sqre%2Finfra%2Fbuild-jupyterlabdemo/activity
+(**TBD**: merge to master and cherry-pick to release branch or merge to release
+branch and merge to ``master``???)
 
-.. code-block::
+Build and Publish
+^^^^^^^^^^^^^^^^^
 
-    WIPEOUT: true
-    MANIFEST_ID: bXXXX
-    COMPILER: devtoolset-6
-    EUPS_TAG: v42_0_0_rc1
-    RELEASE_IMAGE: lsstsqre/centos:7-stack-lsst_distrib-v42_0_0_rc1
-    NO_PUSH: false
+Run the jenkins `release/official-release <official-release>`_ job.
 
-Run ``validate_drp``
-^^^^^^^^^^^^^^^^^^^^
+For input source git refs use the previous ``rc`` tag along with the release
+branch(es)t.  **Ensure** that the release branch is specified to the right of
+the ``rc`` in the listing of git refs.
 
-https://ci.lsst.codes/blue/organizations/jenkins/sqre%2Fvalidate_drp/activity
+Example 1:
 
-.. code-block::
+.. code-block:: text
 
-    WIPEOUT: true
-    MANIFEST_ID: bXXXX
-    COMPILER: devtoolset-6
-    EUPS_TAG: v42_0_0_rc1
-    RELEASE_IMAGE: lsstsqre/centos:7-stack-lsst_distrib-v42_0_0_rc1
+   SOURCE_GIT_REFS: v42.0.0.rc1 v42.x
+   RELEASE_GIT_TAG: v42.0.0.rc2
+   O_LATEST: false
 
-2nd+ Release Candidate(s)
--------------------------
+Example 2:
 
-**Any subsequent ``rc` differs slightly from the initial ``rc1`` process
-because it inherently is not identical as a previous ``git tag`` (if it was,
-there would be no reason to produce another ``rc``). The creation of a git
-release branch prior to ``rc1`` would eliminate the differences.**
+.. code-block:: text
 
-Branch, Merge, Tag
-^^^^^^^^^^^^^^^^^^
+   SOURCE_GIT_REFS: v42.0.0.rc5 v42.x
```

Comment:
The precedence here is flipped.  The branch name should be left-most.

Reply:
OK

Reply:
Fixed.